data/games/strategy/triplea/help/tripleaForum.html

```diff
@@ -0,0 +1,14 @@
+<html>
+	<body>
+		<h1>Posts to forums.triplea-game.org</h1>
+		This poster is build for PBEM games via forums.triplea-game.org.<br>
+		<h2>Instructions:</h2>
+		<ol>
+			<li>Create a new Forum post in the Play by Forum category https://forums.triplea-game.org/category/6/</li>
+			<li>Copy the topic id from the URL displayed in the address bar of you browser. If the URL is https://forums.triplea-game.org/topic/24/ put the topic number (24) into the Topic ID field in TripleA</li>
+			<li>Put your login and password for the forums.triplea-game.org forum into the login and password fields</li>
+			<li>Click the Test Post button, to check that TripleA can reply to your forum post</li>
+		</ol>
+		* Make sure you have registered for receiving dice emails if your dice server sends them out. Click the 'Help?' button by your dice server to see if this is required.<br>
```

Comment:
A link/steps for how to register for receiving dice emails will make these instructions more helpful.